src/operations.c

```diff
@@ -16,7 +16,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  * @file operation.c
- * @author Keith James <kdj@sanger.ac.uk>
+ * @author Keith James <kdj@sanger.ac.uk>, Rob Davies <rmd@sanger.ac.uk>
```

Comment:
Not sure if you need to add me as an author here...

Reply:
I default to including contribs from outside core team - let me know if you'd prefer not.